include/Camera/Camera.hpp

```diff
@@ -129,7 +129,7 @@ class CPolarSubCamera : JDrama::TLookAtCamera {
 	s16 getUnk2C8() const { return unk2C8; }
 
 public:
-	s32 unk50;
+	int mMode;
 	char filler1[0x2C8 - 0x54];
 	s16 unk2C8;
```

Comment:
please add `/* 0x50 */` etc comments before fields whenever you see them missing, they are very useful to quickly look up the field you want while decomping